spec/griddler/cloudmailin/adapter_spec.rb

```diff
@@ -1,14 +1,35 @@
 require 'spec_helper'
 
 describe Griddler::Cloudmailin::Adapter, '.normalize_params' do
-  it_should_behave_like 'Griddler adapter', :cloudmailin, {
-    envelope: {
-      to: 'Hello World <hi@example.com>',
-      from: 'There <there@example.com>',
-    },
-    plain: 'hi',
-    headers: { Cc: 'emily@example.com' },
-  }
+  context 'without bcc' do
+    it_should_behave_like 'Griddler adapter', :cloudmailin, {
+      envelope: {
+        to: 'hi@example.com',
+        from: 'there@example.com',
+      },
+      plain: 'hi',
+      headers: {
+        From: 'there@example.com',
+        To: 'Hello World <hi@example.com>',
+        Cc: 'emily@example.com'
+      },
+    }
+  end
+
+  context 'with bcc' do
+    it_should_behave_like 'Griddler adapter', :cloudmailin, {
```

Comment:
Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.